packages/amazonq/.changes/next-release/Bug Fix-44e5bf6c-38ed-4cba-9d88-c74dd102aa18.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Amazon Q Chat: Fixed regression in the markdown table styles"
```

Comment:
Since we haven't released the regression (and won't), we don't need a changelog item.

Reply:
removed